src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -1865,6 +2560,84 @@ void TableDockWidget::writeMascotGeneric(QString filename) {
   file.close();
 }
 
+void TableDockWidget::showRelabelWidget()
+{
+  relabelDialog->showDialog();
+}
+
+void TableDockWidget::relabelGroups(float changedBadLimit, float changedMaybeGoodLimit)
+{
+
+  disconnect(_legend,
+              &MultiSelectComboBox::selectionChanged,
+              this,
+              &TableDockWidget::filterForSelectedLabels);
+  
+  if (changedBadLimit == badGroupLimit 
+      && changedMaybeGoodLimit == maybeGoodGroupLimit) {
+    QMessageBox::warning(this,
+                          "No Change!",
+                          "The curation parameters you entered are same as" 
+                          " in classified table.\nYou may change the range"
+                          " and try again.\n");
```

Comment:
"The curation parameters you entered are the same as they were in the classified peak-table."

"You may change the range and try again."